aiverify-foundation/moonshot-data
# Ms 477 return metric config

## Description

Modified metrics modules to two specific fields `endpoints` and `configurations` from metrics_config.json

## Motivation and Context

To add in the two additional fields so that they can be returned via the API for the requirements and configurations.

## Type of Change

feat: A new feature
## How to Test

1. Checkout the same branch on main Moonshot
2. Launch the CLI and do a `list_metrics`. Every metric should now have the additional fields `Endpoints` and `Configurations`, which are read from the configuration file.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

-

## Additional Notes

[-

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
